src/Compat.jl

```diff
@@ -1122,6 +1122,67 @@ if VERSION < v"1.8.0-DEV.1016"
     export chopprefix, chopsuffix
 end
 
+# https://github.com/JuliaLang/julia/pull/54653: add Fix
+@static if !isdefined(Base, :Fix)
```

Comment:
```suggestion
@static if !isdefined(Base, :Fix) # VERSION < v"1.12.0-DEV.981"
```
(Can leave the condition as is, but having the Julia version noted comes handy once we set about weeding out old stuff.)